crates/table/src/table.rs

```diff
@@ -329,60 +357,110 @@ impl Table {
         &mut self,
         column_schemas: Vec<ColumnSchema>,
     ) -> Result<Vec<ColumnSchema>, Box<ChangeColumnsError>> {
-        /// Validate that the old row type layout can be changed to the new.
-        // Intentionally fail fast rather than combining errors with [`spacetimedb_data_structures::error_stream`]
-        // because we've (at least theoretically) already passed through
-        // `spacetimedb_schema::auto_migrate::ensure_old_ty_upgradable_to_new` to get here,
-        // and that method has proper pretty error reporting with `ErrorStream`.
-        // The error here is for internal debugging.
-        fn validate(
```

Comment:
This has been moved out to `validate_row_type_layout` to be shared with `validate_add_columns_schema`